core/src/main/java/cz/xtf/core/config/HelmConfig.java

```diff
@@ -0,0 +1,26 @@
+package cz.xtf.core.config;
+
+import java.nio.file.Paths;
+
+public class HelmConfig {
+
+    public static final String HELM_BINARY_CACHE_PATH = "xtf.helm.binary.cache.path";
+    public static final String HELM_BINARY_CACHE_ENABLED = "xtf.helm.binary.cache.enabled";
+    public static final String HELM_BINARY_PATH = "xtf.helm.binary.path";
+    public static final String HELM_CLIENT_VERSION = "xtf.helm.client.version";
```

Comment:
So for consistency can you provide also method for `HELM_CLIENT_VERSION` and use that method in rest of MR?